src/InputWithLabel.js

```diff
@@ -0,0 +1,23 @@
+import React, { useRef, useEffect } from "react";
+
+function InputWithLabel(props) {
+    const inputRef = useRef();
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
```

Comment:
I had a comment about this that I deleted because they actually want you to have a `useEffect` without a dependency array so that the useEffect will trigger anytime a change happens. I do not recommend this in future projects because the useEffect will trigger after every render. There's a better way of doing this but since the app is small we can just remove the dependency array.